README.rst

```diff
@@ -111,7 +111,25 @@ within MATLAB by typing:
 
     >> installDevTools()
 
-|bulb| Check out the `MATLAB.devTools <https://github.com/opencobra/MATLAB.devTools>`__ - and contribute the smart way!
+|bulb| Check out the `MATLAB.devTools
+<https://github.com/opencobra/MATLAB.devTools>`__ - and contribute the smart
+way! The **official documentation** is `here <https://opencobra.github.com/MATLAB.devTools>`__.
```

Comment:
this link <https://opencobra.github.com/MATLAB.devTools> (at least at the moment) does not work. 
I guess it should be:
in any case this should be (I think) https://opencobra.github.io/MATLAB.devTools, but even that link does not work.

Reply:
Once the regular merge on the MATLAB.devTools has been made, these links will work. We are aware of that.